vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -194,9 +194,10 @@ std::string MediaReddit::GetUserId(const std::string& response) {
   if (response.empty()) {
     return std::string();
   }
-
+  std::string pattern(ExtractData(response, "hideFromRobots\":",
```

Comment:
could you please use `std::string pattern =` and const

Reply:
Done